plone/app/caching/purge.py

```diff
@@ -208,6 +208,7 @@ def schemas():
                     continue
                 filename = value.filename
                 if is_image:
+                    yield f"{prefix}/@@images/image"
```

Comment:
```suggestion
                    yield f"{prefix}/@@images/{fieldname}"
```
The image can be stored in a differently named field.

Reply:
@jensens 
```suggestion
                    yield f"{prefix}/@@images/{field_name}"
```
You are right but I think you missed an `_` in field_name